docs/extensions-plugins/jupyterlab.mdx

```diff
@@ -59,24 +59,27 @@ _The <a target="_blank" href="{{ links.website.pfd_desktop_install }}">Pieces Su
 
 ---
 
-### Install Using Pip
-
+<Accordion title="Install Using Pip">
 1. Run this command:
-    ```pip install -U jupyter-pieces```
+    ```sh
+    pip install -U jupyter-pieces 
+    ```
+2. *(optional)* Be Sure to have extensions enabled. [Read about enabling Extensions](https://tljh.jupyter.org/en/latest/howto/admin/enable-extensions.html)
 
-2. (optional) Be Sure to have extensions enabled. [Read about enabling Extensions here](https://tljh.jupyter.org/en/latest/howto/admin/enable-extensions.html)
-    - Be sure to accept the disclaimer before moving forward in this process.
+<Info>Be sure to accept the disclaimer before moving forward in this process.</Info>
+</Accordion>
 
```

Comment:
if you add the `<Spacer></Spacer>` here with an empty line above and below, you will get the spacer addition that you are looking for. These variables are defined in the head of the file so if you go up to `export const Spacer` you can see the properties of that specific spacer there. There is also an example of the spacer being used on line 80 (line 83 in the updated file) that you can copy and paste there as well, but note that those spacers share the properties in the head and if you change the value of the spacer in its definition, it will effect both locations.

Reply:
I went ahead and updated this - we can talk more about it soon